### PR TITLE
Shim script generator and vscode instructions

### PR DESCRIPTION
**Pull request by wadetandy**
Reopening from #201 after the rebase.  This adds a new script for generating an executable shim script as opposed to the currently recommended alias, as well as README instructions for setting up vscode. Does not include README update for the shim script, but glad to update with that if you want to make it the path going forward.

### CODE REVIEW

File: script/install_shim.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+if [ $# -eq 0 ]
+  then
+    echo "Please pass a file location to install the shim script"
+    exit 1
+fi
+
+echo "#!/usr/bin/env bash
+export RUBYFMT_USE_RELEASE=1
+$(which ruby) --disable=all $(pwd)/rubyfmt.rb \$@" > "$1"
+chmod +x "$1"
```
Comment (fables-tales) [inline script/install_shim.sh]: could you add a trailing newline here?

File: script/install_shim.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```
Comment (fables-tales) [inline script/install_shim.sh:1]: could you add a test for this shim script?

**Review by fables-tales — CHANGES_REQUESTED** [review]
mostly lgtm, just a few requests.

**Comment by wadetandy** [comment]
@penelopezone Updated with requested changes

File: README.md [hunk 3]
```diff
@@ -52,3 +52,17 @@ e.g.:
 * `RUBYFMT_DISABLE_SZUSH=1`: disables the backend render queue writer,
   very useful for debugging, literally useless if you're not developing rubyfmt
   itself.
+
+## Editor Support
+
+### Visual Studio Code
+
```
Comment (fables-tales) [inline README.md:73]: you should reference the shim script, and then I will merge this.

Comment (wadetandy) [reply README.md:73]: Updated README ✅

**Pull request merged**